task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -482,6 +482,17 @@ export default {
         } else {
           this.taskStartDate = value;
         }
+      } else if (value === 'none' && this.oldTask.startDate) {
+        this.task.startDate = null;
+        this.$taskDrawerApi.updateTask(this.task.id, this.task).then(() => {
+          this.oldTask.startDate = this.task.startDate;
```

Comment:
Is this needed ?

Reply:
cc @hthabti Can you answer to Ali's question ? 
Thanks